src/main/java/core/basesyntax/MatrixOperation.java

```diff
@@ -6,18 +6,42 @@
  */
 public class MatrixOperation {
     public int[] getDiagonal(int[][] matrix) {
-        return null;
+        int[] diadonall = new int[matrix.length];
+        for (int i = 0; i < diadonall.length; i++) {
+            diadonall[i] = matrix[i][i];
+        }
+        return diadonall;
     }
 
     public int[] getCounterDiagonal(int[][] matrix) {
-        return null;
+        int[] sideDiadonal = new int[matrix.length];
```

Comment:
```suggestion
        int[] counterDiagonal = new int[matrix.length];
```
naming should be consistent(if method is getCounterDiagonal, so the array should be named in the same way)